### PR TITLE
docs: add sponsors

### PR DESCRIPTION
**Pull request by LayZeeDK**
## PR Checklist

Please check if your PR fulfills the following requirements:

```
[x] The commit message follows our guidelines: CONTRIBUTING.md#commit
[ ] Tests for the changes have been added (for bug fixes / features)
[ ] Docs have been added / updated (for bug fixes / features)
```

## PR Type

What kind of change does this PR introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[x] Documentation content changes
[ ] Other... Please describe:
```

## What is the current behavior?

Issue Number: N/A

## What is the new behavior?

Add nxworker sponsors.

## Does this PR introduce a breaking change?

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,17 +1,26 @@
 # @nxworker/\* monorepo
 
-Monorepo for the `nxworker` NPM organization. Plugins and utilities for Nx workspaces.
+Monorepo for the `@nxworker` NPM organization. Plugins and utilities for Nx workspaces.
 
-## Wallaby.js
+## Sponsors
```
Comment (NachoVazquez) [inline README.md:5]: Should we apply to the sponsor program of GitHub?

Screen Shot 2021-01-13 at 18 12 03

Comment (LayZeeDK) [reply README.md:5]: Would it benefit you? I don't want to deal with Danish taxes to earn a few bucks or figure out how to be able to spend it on for example graphical design.

Comment (NachoVazquez) [reply README.md:5]: I don't have any motivation related to this. It just seemed a logical question given the creation of the sponsor section.

**Review by NachoVazquez — DISMISSED** [review]

File: README.md (hunk 1 above)
Comment (LayZeeDK) [reply README.md:5]: With these sponsors, we're getting useful gifts without having to deal with any paperwork 😄 Besides, we don't have any features to motivate sponsorship right now 😉

**Review by NachoVazquez — APPROVED** [review]

**Pull request merged**